Xamarin.Forms.Platform.Android/Extensions/RecyclerExtensions.cs

```diff
@@ -0,0 +1,38 @@
+﻿#if __ANDROID_29__
+using AndroidX.AppCompat.Widget;
+using AndroidX.RecyclerView.Widget;
+#else
+using Android.Support.V7.Widget;
+#endif
+using AView = Android.Views.View;
+
+namespace Xamarin.Forms.Platform.Android
+{
+	internal class RecyclerExtensions
```

Comment:
There are no extension methods in here; it shouldn't be named Extensions.

Reply:
fixed